### ISSUE TITLE
Remove tutorial "streets" from auditing route selection

### ISSUE THREAD

**Issue by jonfroehlich** [ProjectSidewalk/SidewalkWebpage#2782]
I recall having an open Issue on this already but I couldn't find it. Sorry @misaugstad!

But now that we are in non-US cities, it's quite startling and disconcerting to suddenly be transported to the tutorial location in DC. This really should not happen. Can we remove any possibility of going to the tutorial streets while doing actual auditing? This seems obvious but I recall the fix being non-trivial.

See video:
https://user-images.githubusercontent.com/1621749/155006923-d6f9fadc-952a-4c86-8207-8ada472ebb90.mp4

**Comment by jonfroehlich** [comment]
This keeps happening to me in Amsterdam :(. After completing one mission in a new neighborhood, the second one is often in DC.

![image](https://user-images.githubusercontent.com/1621749/155546287-85f3262d-b5c7-443d-badc-09464be87c83.png)

**Comment by misaugstad** [comment]
I think that this is doable, though it will take some care in implementation. Our code assumes that each street is associated with a neighborhood, so I'm just kind of associating the tutorial street with some random neighborhood in each city. A ton of stuff would break if we violated that association.

I think that it would be easier to just choose not to route users down that street on the front-end. However, this requires back-end updates as well. Anywhere where we are counting the streets/distance left to audit in the neighborhood, etc., would need to be updated to filter our the tutorial street. We don't want the maps to say that there's a little bit left to audit when that isn't true. And we don't want the front-end to think there's another street to audit and then have us ending up in a bad state.

**Comment by jonfroehlich** [comment]
OK, this just happened again in Taiwan:

> Hi, one of our auditor was directed to different country while auditing New Taipei City. Can you check if the issue is from Project Sidewalk Platform or from Google? Thanks

![image](https://github.com/ProjectSidewalk/SidewalkWebpage/assets/1621749/f70e74f6-cad0-46cc-bbc8-98957a674080)

**Comment by jonfroehlich** [comment]
It really seems like we should be able to address this. Shouldn't we special case the tutorial? Maybe even build a separate database table to handle it?

**Comment by misaugstad** [comment]
Yeah we can totally figure this out. Over the years I've made some simplifications to the front end, and I understand way more about it now (so I have more of an idea of what might break). Will it take 5 `if` statements? 10? 15..? Would a tiny restructure bring it from 15 back down to 5? Unclear, but I feel more confident that I can make it happen without something breaking.
